frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -222,30 +214,7 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
     </>
   );
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/ban-ts-ignore
-  // @ts-ignore
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const operandsSection = operandsLoadedErrorMessage ? (
```

Comment:
We're about to reenable this, so we shouldn't delete the code. Let's comment out instead.